int16 leftPWM
int16 rightPWM